### ISSUE TITLE
LCA in R notebook on Jupyter

### ISSUE THREAD

**Issue by datawookie** [dominodatalab/low-code-assistant#28]
This might well be a known issue, but if I create an R notebook in Jupyter and then press the LCA button I get the Python started code.

![image](https://user-images.githubusercontent.com/6134409/199701643-6e5b6307-d213-410f-9f47-8825550cbea9.png)

**Comment by mariobuikhuizen** [comment]
Good point. We always expect python in the notebook :blush:. I don't know if it's possible to change the toolbar buttons based on the kernel used, though.
